apps/fluent-tester/src/TestComponents/Chip/ChipTest.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+
+import { ChipDefault } from './ChipDefault';
+import { E2EChipTest } from './E2EChipTest';
+import { CHIP_TESTPAGE } from '../../../../E2E/src/Chip/consts';
+import type { TestSection, PlatformStatus } from '../Test';
+import { Test } from '../Test';
+
+const chipSections: TestSection[] = [
+  {
+    name: 'Default Chip',
+    testID: CHIP_TESTPAGE,
+    component: ChipDefault,
+  },
+];
+
+const e2eSections: TestSection[] = [
+  {
+    name: 'Chip E2E',
+    component: E2EChipTest,
+  },
+];
+
+export const ChipTest: React.FunctionComponent = () => {
+  const status: PlatformStatus = {
+    win32Status: 'Backlog',
+    uwpStatus: 'Backlog',
+    iosStatus: 'Backlog',
+    macosStatus: 'Backlog',
+    androidStatus: 'Experimental',
+  };
+
+  const description =
+    'Chips are compact representations of entities (most commonly, people) that can be typed in, deleted or dragged easily. The most common example can be seen in the To field of a new email in Outlook Mobile.';
```

Comment:
i think we can remove the reference used like outlook mobile. We want give detail about only the component here.

Reply:
Removed